bin/release_android

```diff
@@ -17,22 +17,41 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
-USER=$2
-if [ -z "$USER" ]; then
-    print_error "Missing Bintray API user"
-    exit 1
-fi
+URL="https://oss.sonatype.org/service/local/staging/deploy/maven2"
 
-TOKEN=$3
-if [ -z "$TOKEN" ]; then
-    print_error "Missing Bintray API token"
-    exit 1
+if [ "$VERSION" == "0.0.0-dev" ]; then
+    VERSION="0.0.0-SNAPSHOT"
+    URL=https://oss.sonatype.org/content/repositories/snapshots
 fi
 
-sed -i -E "s|<version>.+</version>|<version>$VERSION</version>|g" bin/package/android/pom.xml
+sed -i -E "s|<version>.+</version>|<version>$VERSION</version>|g" bin/package/android/mvn.pom
 
-URL=https://api.bintray.com/content/mysterium/Node/mobile-node/$VERSION/mysterium/network/mobile-node/$VERSION
+mvn gpg:sign-and-deploy-file -e -X \
+    -settings=bin/package/android/mvn.settings \
+    -Durl=$URL \
+    -DrepositoryId=ossrh \
+    -Dgpg.keyname=336826E2222A0B29 \
+    -DpomFile=bin/package/android/mvn.pom \
+    -Dfile=build/package/Mysterium.aar
+
+if [ "$VERSION" == "0.0.0-SNAPSHOT" ]; then
+    exit 0
```

Comment:
Lets validate VERSION there it's parse from injected arguments

Reply:
Do we really? Its CI script not exposed to any running service.

Reply:
I didn't get it, what should be validated?

Reply:
I meant weird exit of script in the middle of script + no message. Would be nice improve